### ISSUE TITLE
Error while tranining model with or without checkpoint

### ISSUE THREAD

**Issue by cgarg-tud** [fuy34/planerecover#9]
Hi I have been trying to run the code but I am facing some issues while training data. I downloaded the datasets for training from zip folder link. Then I try to run it. I also tried it running with a checkpoint provided. I still see the same messagesee the message in terminal as shown in first image. The training resumes but when i check the tensorboard i see no processing of images. as shown in second image. Should I just wait and its actually right. 

![image](https://user-images.githubusercontent.com/45408048/66677371-bf7ef300-ec69-11e9-8144-24b9677dce7a.png)

![image](https://user-images.githubusercontent.com/45408048/66676927-e2f56e00-ec68-11e9-9525-d80e5e24a568.png)

**Comment by fuy34** [comment]
Hi, I am not really sure what happened there. 
According to your current setting, the code will write the log information into tensorboard every 1000 steps, and there will be a message pops out in our first terminal indicating the training loss. 

Did you see that message when you checked the tensorboard in the second terminal? 
Depending on the machine, it might take minutes to reach 1000 steps. 

If there is still nothing in tensorboard after 1000 steps, please let me know.

**Comment by cgarg-tud** [comment]
Thankyou for your reply.  I waited for more time and I got the message that you are talking about.

**Comment by cgarg-tud** [comment]
Also I tried using the testing command. I received the following error in the image provided below. The thing is that the downloaded data has no 'images' folder inside as such. 29 folder has directly images and camera parameters and so on. So may be I am missing something or I should change the path in test_synthia file somewhere

![image](https://user-images.githubusercontent.com/45408048/66703992-6b7f1780-ed18-11e9-8667-745994ad7d19.png)

**Comment by fuy34** [comment]
yes. As we mentioned in readme, we use the original full-resolution data for test, while the data we uploaded is pre-processed data. So the path may be different.  For test, you probably need to modify the path accordingly. 
By the way, you may want to remove seq 22 from the test list, as it is a sequence we intentionally remove from the training data, to test the model performance in unknown sequence. 
See Issue 4 ```https://github.com/fuy34/planerecover/issues/4``` for more details.

**Comment by cgarg-tud** [comment]
Thankyou again. I will check this out as well. I was actually getting lot of errors today but finally the model got trained for 2000 steps.

**Comment by fuy34** [comment]
congrats! Let us know if there is any further question.

**Comment by cgarg-tud** [comment]
Hello, I was continuing from previous step and testing the data in test list of synthia dumped folder. I was able to create the depth map, plane_segments and plane_segments_vis. Then when I further went on to create 3d output by running generate3d.py. I changed the paths for all corresponding variables. I am receiving a strange error provided in image below . For clarification the dataset remains the same as test list and corresponding predicted depth images are used for depth list. I have tried figuring it out. I also printed the img variable to check if there was no image but it printed the image. 
I have also attached the edited generate3d just for reference. 

https://drive.google.com/drive/folders/1kpGwNncqXTp6z6mMjLkHlKV4EeysR0LD?usp=sharing
![image](https://user-images.githubusercontent.com/45408048/67219640-abd14a80-f428-11e9-964d-e4bc44040f61.png)

**Comment by fuy34** [comment]
Hi @cgarg-tud,
Sorry for the late reply. I hope you are doing well. 
I revisited the code again. I realized the python-opencv has been upgraded last year, and the order of arguments seems be changed. 
I have updated the code accordingly, and a new arguments  ```--use_preprocessed``` has been added to better support people who directly use our pre-processed dataset for test. Please refer to the updated README for more details. Thanks.
